interop-tests/src/test/scala/akka/grpc/scaladsl/CountingGreeterServiceImpl.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.scaladsl
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import scala.concurrent.Future
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+
+import example.myapp.helloworld.grpc.helloworld._
+
+class CountingGreeterServiceImpl extends GreeterService {
```

Comment:
Is this one covering the bug?

Reply:
(no, sorry, that was a cleanup I made while reproducing the bug in a testcase, but I ended up testing it with another project..)